docs/README.md

```diff
@@ -1,30 +1,162 @@
-# WORK IN PROGRESS
-
-=� =� =�
-
 # VTEX Seller Selector
 
-=� Don't fork this project. Use, [contribute](https://github.com/vtex-apps/awesome-io#contributing), or open issues through [Store Discussion](https://github.com/vtex-apps/store-discussion).
-
-## Description:
+![exampleLayout](https://user-images.githubusercontent.com/53904010/78715148-595eea00-78f3-11ea-90fa-dc38c37a80d8.png)
 
 Seller Selector enables your visitors to easily see how many sellers your marketplace has for certain products, check their prices
 and add then to their cart.
 
-## Preview:
+## Configuration:
+
+1. Import the Seller Selector app to your theme's dependencies in the `manifest.json` as shown below:
+
+```json
+"dependencies": {
+  "vtex.sellerselector": "0.x"
+}
+```
+
+2. Add the `link-seller` block inside your product page, this will enable a link to the Seller Selector main page, for example:
+
+```json
+"flex-layout.col#right-col": {
+    "props": {
+      "preventVerticalStretch": true,
+      "rowGap": 0
+    },
+    "children": [
+      "product-name",
+      "link-seller"
+      "product-price#product-details",
+      "product-separator",
+      "flex-layout.row#buy-button",
+      "availability-subscriber",
+      "shipping-simulator",
+      "share#default"
+    ]
+  },
+```
+3. The page cointaining all sellers will be displayed automatically, but you can configure the page layout and props for each block.
+
+# Advanced Configuration
+According to the Seller Selector composition, it can be highly customizable using it's specific blocks. Currently, its default implementation is as follows:
+
+```json
+{
+  "store.sellers": {
+    "blocks": [
+      "seller-table"
+    ]
+  },
+  "seller-table": {
+    "children": [
+      "vtex.store-components:product-name",
+      "seller-simulate-shipping",
+      "seller-head",
+      "seller-body"
+    ]
+  },
+  "seller-head": {
+    "children": [
+      "seller-head-cell#Seller",
+      "seller-head-cell#Price",
+      "seller-head-cell#Shipping",
+      "seller-head-cell#PriceWithShipping",
+      "seller-head-cell#BuyButton"
+    ]
+  },
+  "seller-head-cell#Seller": {
+    "props": {
+      "title": "Seller"
+    }
+  },
+  "seller-head-cell#Price": {
+    "props": {
+      "title": "Price"
+    }
+  },
+  "seller-head-cell#Shipping": {
+    "props": {
+      "title": "Shipping"
+    }
+  },
+  "seller-head-cell#PriceWithShipping": {
+    "props": {
+      "title": "Price With Shipping"
+    }
+  },
+  "seller-head-cell#BuyButton": {
+    "props": {
+      "title": "Add To Cart"
+    }
+  },
+  "seller-body": {
+    "children": [
+      "seller-row"
+    ]
+  },
+  "seller-row": {
+    "children": [
+      "seller-name",
+      "seller-price",
+      "seller-shipping",
+      "seller-price-with-shipping",
+      "seller-add-to-cart"
+    ]
+  }
+}
+```
+# Props Configuration
```

Comment:
```suggestion
After presenting the default implementation, we should have a table with all blocks used to build the page and a description to clarify their functionality. You can use the Search Results documentation (https://vtex.io/docs/components/all/vtex.search-result@3.54.0/) as an example.  

Also, we have a bunch of blocks with no props. Seller Name, Seller Price, Seller Shipping. Could we have more details about them?
```

Reply:
I've added some descriptions for each block.